fake-files/output-expected/linnarsson/linnarsson.cell_sets.json

```diff
@@ -0,0 +1 @@
61", "196", "1238", "1283", "2886", "1515", "42", "1344", "14", "1547", "2809", "6306", "1635", "401", "3338", "4207", "233", "902", "2423", "4731", "3358", "1385", "2104", "119", "2164", "5308", "1113", "3956", "155", "4511", "1244", "4069", "1534", "152", "2228", "1479", "861", "3076", "92", "4036", "1623", "3419", "950", "844", "297", "2186", "884", "5905", "4098", "1568", "4512", "2768"]}, {"name": "Pericytes", "set": ["5126", "1554", "3672", "3680", "1748", "4434", "3882", "3757", "576", "47", "6076", "2420", "192", "5467", "592", "874", "1014", "1567", "175", "6047", "4132", "2773", "4906", "3361", "2217", "288", "4396", "6168", "3333", "5669", "1767", "4302", "6487", "4313", "197", "4828", "176", "103", "6466", "382", "6327", "52", "1604", "907", "3626", "1752", "6009", "4557", "851", "3376", "855", "1928", "6564", "564", "5484", "4921", "4747", "344", "5027", "3342", "879", "6546", "3733", "5555", "2170", "5538", "5457", "114", "3629", "5904", "6469", "4383", "98", "371", "4881", "105", "1661", "5037", "3724", "380", "2216", "6578", "2615", "2142", "4278", "5034", "1940", "3673", "4246", "6756", "3690", "4550", "4066", "6", "6147", "6150", "520", "302", "877", "599", "7080", "2225", "4526", "1676", "3310", "660"]}, {"name": "Vascular Smooth Muscle", "set": ["3043", "4521", "5341", "2832", "3268", "650", "3034", "4212", "343", "85", "4520", "1313", "1045", "6486", "2883", "3608", "3863", "3031", "2868", "1707", "7641", "4478", "3337", "4437", "1790", "4265", "1064", "2877", "2602", "2940", "3635", "3336", "6462", "260", "4446", "6431", "5462"]}]}, {"name": "Ventricle", "children": [{"name": "C. Plexus", "set": ["3792", "3156", "3177", "1743", "4771", "1701", "3381", "3141", "5749", "2241", "2792", "3155", "97", "6580", "113", "3731", "3169", "1601", "5187", "906", "2782", "3143", "1613", "4746", "5657", "334", "1843", "614", "5583", "229", "3797", "1800", "4530", "4415", "972", "2365", "3625", "4405", "337", "54", "1742", "3423", "3172", "1698", "7825", "1716", "3186", "6586", "5152", "2966", "3159", "290", "6541", "2316"]}, {"name": "Ependymal", "set": ["3184", "3146", "2707", "1201", "3107", "1087", "3129", "1181", "1191", "5366", "3126", "3182", "3151", "4411", "1182", "1179", "3198", "3218", "2604", "1383", "3115", "2788", "1189", "2704", "2896", "2901", "3140", "2663", "5371", "3122", "3138", "3210", "2713", "1202", "4277", "3176", "4185", "1194", "5455", "3167", "1192", "3104", "3201", "3188", "3171", "2902", "2928", "3127", "3202", "3205", "5630", "3112", "3137", "3183", "3211", "1498", "3217", "1200", "1186", "3135", "3161", "3175", "7067", "2924", "3118", "1184", "3130", "3134", "3121", "2722", "2899", "2654", "2720", "3105", "3136", "4262", "3133", "1190", "1185", "4266", "2652", "6595", "3124", "3204", "3132", "1196", "3148", "2716", "1195", "3152", "3199", "3200", "3123", "3147", "2965", "3187", "3120", "3180", "3131", "3220", "1090", "3179", "3209", "3178", "5398", "5408", "2648", "4256", "3197", "1382", "5397", "1381"]}]}]}]}
```

Comment:
Why is this here now?  I don't mind that it's not really a "unit" test but I'm just curious.

Reply:
This follows the current convention of the Linnarsson files in the fake-files folder actually being real files. Not sure why either https://github.com/hubmapconsortium/vitessce-data/tree/master/fake-files/output-expected/linnarsson

Reply:
I'm not sure I really get the question or the answer... ping me if I should?

Reply:
but:
- please pretty print this.
- Consider making it smaller: start with a smaller test input.... if there is a difference in the output, you want the diff to be readable.

Reply:
I have added pretty printing here.

To clarify my previous answer:
This is a new test for the `linnarsson` dataset.

The existing tests for all of the Linnarsson data in this vitessce-data repository use the FULL linnarsson.cells.loom file. https://github.com/hubmapconsortium/vitessce-data/tree/master/fake-files/input/linnarsson

To change the size of this test fixture would mean changing all of the other test fixtures for all of the other linnarsson tests. Should I try to resize the original loom file?